.github/workflows/.reusable-sca.yml

```diff
@@ -40,59 +40,41 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - name: Run
-        uses: ./.github/actions/trivy-image
-        with:
-          image: ${{ inputs.image }}
-          registry: ${{ inputs.registry }}
-          repo_owner: ${{ inputs.repo_owner }}
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          output: ${{ inputs.output }}
-
-  grype:
-    name: grype
-    runs-on: ubuntu-latest
-    if: |
-      inputs.skip != 'non-required' &&
-      inputs.skip != 'all'
-    permissions:
-      packages: read
-      security-events: write
-    container:
-      image: docker:stable
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - name: Run
-        uses: ./.github/actions/grype
-        with:
-          image: ${{ inputs.image }}
-          registry: ${{ inputs.registry }}
-          repo_owner: ${{ inputs.repo_owner }}
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          output: ${{ inputs.output }}
-
-# WIP: Syft issue seems to cause error (https://github.com/anchore/syft/issues/1622)
-  dependency-submission:
-    name: syft / dependency review
-    runs-on: ubuntu-latest
-    if: |
-      inputs.skip != 'non-required' &&
-      inputs.skip != 'all'
-    permissions:
-      packages: read
-      contents: write
-    steps:
       - name: Login with registry
         if: inputs.registry != ''
         uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
         with:
           registry: ${{ inputs.registry }}
           username: ${{ inputs.repo_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Run
-        uses: anchore/sbom-action@55dc4ee22412511ee8c3142cbea40418e6cec693 # v0.17.8
+      - name: Create reports folder
+        run: |
+          mkdir reports
+        shell: bash
+      - name: Run Trivy on image
```

Comment:
For the other MR, you merged two steps doing the same for different inputs. Why not here, too? If merged, you could also ignore my other comment on this